dev/bin/interaction/get_all_bridges.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+source "common.sh"
+
+http -v $MANAGER_URL/api/v1/bridges Authorization:"$OB_TOKEN"
```

Comment:
why not a curl command ?
`http` is not installed by default on Fedora for example

Reply:
You're right I didn't notice